mpu6050/mpu6050.c

```diff
@@ -26,17 +29,23 @@ static int mpu6050_read_data(void)
 		return -ENODEV;
 
 	/* accel */
-	g_mpu6050_data.accel_values[0] = (s16)((u16)i2c_smbus_read_word_swapped(drv_client, REG_ACCEL_XOUT_H));
-	g_mpu6050_data.accel_values[1] = (s16)((u16)i2c_smbus_read_word_swapped(drv_client, REG_ACCEL_YOUT_H));
-	g_mpu6050_data.accel_values[2] = (s16)((u16)i2c_smbus_read_word_swapped(drv_client, REG_ACCEL_ZOUT_H));
+	g_mpu6050_data.accel_values[0] =
+		(s16)(RD_WORD_SW(drv_client, REG_ACCEL_XOUT_H));
```

Comment:
This macros is always used for return signed value. May be it is better to include this conversation in macros.

Reply:
Included in the macro.